draft-ietf-tls-esni.md

```diff
@@ -591,10 +591,10 @@ ClientHello extensions can be computed in this way.
 In contrast, clients do not know the longest SNI value in the client-facing
 server's anonymity set without server input. For the "server_name" extension
 with length D, clients SHOULD use the server's length hint L
-(ECHCOnfig.maximum_name_length) when computing the padding as follows:
+(ECHConfigContents.maximum_name_length) when computing the padding as follows:
```

Comment:
Is `ECHConfig.config.maximum_name_length` more appropriate here?

Reply:
Seems reasonable, done.